nats_test.go

```diff
@@ -1434,6 +1434,51 @@ func TestExpiredUserCredentials(t *testing.T) {
 	chainedFile := createTmpFile(t, creds)
 	defer os.Remove(chainedFile)
 
+	ch := make(chan bool)
+
+	url := fmt.Sprintf("nats://127.0.0.1:%d", TEST_PORT)
+	nc, err := Connect(url,
+		UserCredentials(chainedFile),
+		ReconnectWait(25*time.Millisecond),
+		MaxReconnects(2),
```

Comment:
Change MaxReconnects() here to higher or even infinite (-1) to prove that we close due to repeated error

Reply:
Good point, will fix.